platform/golden/scripts/serial.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+cat > /tmp/fixes.sh <<EOF
+#!/bin/bash
+set -x
+echo 'GRUB_CMDLINE_LINUX="console=ttyS0"' >> /etc/default/grub
```

Comment:
better to put in /etc/default/grub.d. I will submit an update.

Reply:
This _may_ not even work, still trying to get a testable environment for dev  (I think I can do this in OCI now, but I haven't yet managed to get a connection through the UI or via `oci` cli yet).